app/controllers/api/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::PostsController < ApiController
+  def index
+    @user = User.find(params[:user_id])
+  end
```

Comment:
**OPTIONAL** - You have done well so far, but according to the project requirements, this controller should return all posts belonging to a particular user. Please fix this 🙏